lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -466,6 +466,13 @@ spec = do
             , "                             trezor (12, 15, 18, 21 or 24 words)"
             , "                             ledger (12, 15, 18, 21 or 24 words)"
             ]
+        ["key", "public", "--help"] `shouldShowUsage`
+            [ "Usage:  key public HEX-XPRV"
```

Comment:
*XPRV

The meta-type shouldn't mix the purpose with the encoding. (The meta-type says _what_ is expected, not _how_).

Reply:
Fixing in #1370